core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -170,6 +170,10 @@ private void registerHttpHostAllowListHandler(final OpenAPI3RouterFactory openAp
   }
 
   private void registerSwaggerUIRoute(final Router router) throws IOException {
+    if (!config.isSwaggerUIEnabled()) {
```

Comment:
nit: would rather not even call registerSwaggerUIRoute if isSwaggerUIEnabled is false. i.e. don't call this method in the run is swagger ui isn't enabled.

Reply:
done, moved it out of method.